packages/kit/src/views/BulkCopyAddresses/pages/BulkCopyAddresses.tsx

```
@@ -1,7 +1,7 @@
 import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 
 import BigNumber from 'bignumber.js';
-import { flatten, groupBy, isEmpty, isNaN, map } from 'lodash';
+import { flatten, groupBy, isEmpty, isNaN, isNil, map } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)